lib/data/service/network_service/dio_network_service_impl.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:askm/core/typedefs.dart';
```

Comment:
<img width="386" alt="image" src="https://github.com/benikgalstyan/askm/assets/105119340/4c4d67c0-5135-4208-b5cd-8aba631e37aa">

add patch method also 

patch similar with put 

differents:  
put - implies a complete change in resources

patch - partial change of resource